docs/docs/exosphere/state-manager-setup.md

```
@@ -0,0 +1,374 @@
+## Monitoring and Health Checks
+
+### Health Check Endpoint
+
+The state manager provides a health check endpoint:
+
+```bash
+curl http://localhost:8000/health
+```
+
+Response:
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)